.versionrc.json

```diff
@@ -0,0 +1,50 @@
+{
+  "tagPrefix": "",
```

Comment:
In order to not publish a `1.0.0` version with the next release, you will need to add the `"preMajor": true` option here.

Reply:
Didn't know that, I'll have to change this in some other repos as well, thanks!